packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -203,21 +203,54 @@ export class MetricExporter implements PushMetricExporter {
   }
 
   /**
-   * Calls CreateMetricDescriptor in the GCM API for the given InstrumentDescriptor
+   * Returns true if a descriptor already exists within the requested GCP project id;
+   * @param descriptor The metric descriptor to check
+   * @param projectIdPath The GCP project id path
+   * @param authClient The authenticated client which will be used to make the request
+   * @returns {boolean}
+   */
+  private async checkIfDescriptorExists(
```

Comment:
nit so that non TypeScript users know this is private
```suggestion
  private async _checkIfDescriptorExists(
```

Reply:
Sure, addressed @aabmass.